src/modules/hip/kernel/crop_mirror_normalize.hpp

```diff
@@ -89,66 +106,78 @@ __global__ void crop_mirror_normalize_pln_tensor(T *srcPtr,
     }
 
     uint srcIdx;
-    d_float8 pix_f8, cmnParams_f8;
     uint dstIdx = (id_z * dstStridesNCH.x) + (id_y * dstStridesNCH.z) + id_x;
-
-    cmnParams_f8.f4[0] = (float4)meanTensor[id_z];
-    cmnParams_f8.f4[1] = (float4)(1 / stdDevTensor[id_z]);
+    int incrementPerImage = id_z * channelsDst;
+    d_float8 pix_f8, cmnParams_f8;
+    cmnParams_f8.f4[0] = (float4)meanTensor[incrementPerImage];          // Get mean for R channel
+    cmnParams_f8.f4[1] = (float4)(1 / stdDevTensor[incrementPerImage]);  // Get (1 / stdDev) for R channel
```

Comment:
it is better to calculate this at higher level and pass 1/std value to the kernel

Reply:
1/stdDev has been moved out now